packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -32,19 +54,19 @@ const DetailComponent = ({
     is_mt5,
     handlePOIforMT5Complete,
     ...props
-}) => {
-    const [status, setStatus] = React.useState();
-    const [response_error, setError] = React.useState();
+}: TDetailComponent) => {
+    const [status, setStatus] = React.useState<string>();
+    const [response_error, setError] = React.useState<string | undefined>();
```

Comment:
just a suggestion: maybe it is better to have here default empty line, that undefinied. 
Otherwise we will have to set error prop for UnsupportedFailed as optional